src/main/java/com/opencart/pages/HeaderPage.java

```diff
@@ -48,6 +48,19 @@
     @FindBy(css = "button[onclick=\"wishlist.add('43');\"]")
     private WebElement macbookWishListButton;
 
+    @FindBy(css = "a[title = 'Shopping Cart'] span")
+    private WebElement shoppingCartSpan;
+
+    @FindBy(css = "span#cart-total")
+    private WebElement shoppingCartDropDownInfo;
+
+    @FindBy(css = "p.text-center")
+    private WebElement shoppingCartDropDownMessage;
+
+    @FindBy(css = "button.btn.btn-inverse.btn-block.btn-lg.dropdown-toggle")
+    private WebElement shoppingCartDropDownButton;
```

Comment:
you have the possibility to use id here, try to use the simplest selectors, 
try with smth like this: '#cart .btn'